modules/katello_content_view.py

```diff
@@ -97,9 +98,7 @@ def find_organization(self, name):
             self._module.fail_json(msg="No organization found for %s" % name)
 
     def find_product(self, name, organization):
-        org = self.find_organization(organization)
```

Comment:
No, we still use it on [line 124](https://github.com/theforeman/foreman-ansible-modules/pull/7/files#diff-ad26508e5ed85293d1427174dd4abc74R124).